decidim-debates/db/migrate/20200708072042_fix_debates_i18n_fields.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class FixDebatesI18nFields < ActiveRecord::Migration[5.2]
+  class User < ApplicationRecord
+    self.table_name = :decidim_users
+  end
+
+  class Debate < ApplicationRecord
+    self.table_name = :decidim_debates_debates
+  end
+
+  class Organization < ApplicationRecord
+    self.table_name = :decidim_organizations
+  end
+
+  def change
+    reset_column_information
+
+    Debate.find_each do |debate|
+      next if debate.decidim_author_type == "Decidim::Organization"
+
+      locale, org_id = User.where(id: debate.decidim_author_id).pluck(:locale, :decidim_organization_id).first
+      locale = locale.presence || Organization.find(org_id).default_locale
+      locale = locale.to_s
+
+      debate.title = {
+        locale => debate.title[locale]
+      }
+      debate.description = {
+        locale => debate.description[locale]
+      }
+
+      debate.save!
```

Comment:
Should we skip validations? Maybe there are invalid debates (for whatever reasons) and we need to update the values of the contents but we don't care about the rest.